docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.9'
+services:
+  db:
+    image: mysql
+    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: Do.Test.Blueprints.Service.Application
+      MYSQL_TCP_PORT: 3366
+    ports:
+      - 3366:3366
+    healthcheck:
+      test: ["CMD", 'mysqladmin', 'ping', '-h', 'localhost','-P','3366', '-u', 'root', '-p$$MYSQL_ROOT_PASSWORD' ]
+      start_period: 2s
+      interval: 5s
+      timeout: 5s
+      retries: 5
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin
+    depends_on:
+      db:
+        condition: service_healthy
+    environment:
+      PMA_HOST: db
+      PMA_PORT: 3366
+    restart: always
+    ports:
+      - 8080:80
+  do.test.blueprint.service.application:
+    build:
+      dockerfile: test/blueprints/Do.Test.Blueprints.Service.Application/Dockerfile
+      args: 
+        ENVIRONMENT: Production
+    ports:
+      - 5007:80
```

Comment:
port should be the same as current launch profile of blueprints service application